backend/de.metas.business/src/main/java-legacy/org/compiere/model/MInvoice.java

```diff
@@ -1018,6 +1020,12 @@ public boolean calculateTaxTotal()
 				}
 			}
 		}
+
+		if(apply5CentRounding())
+		{
+			grandTotal = NumberUtils.roundTo5Cent(grandTotal);
+		}
```

Comment:
not sure if here is the best place... but might be that we have to check/change org.compiere.model.MInvoiceLine#updateHeaderTax too.